packages/lab/stories/tooltip.stories.tsx

```diff
@@ -1,65 +1,110 @@
-import { Button, useForkRef } from "@salt-ds/core";
-import {
-  Tooltip,
-  TooltipProps,
-  useTooltip,
-  UseTooltipProps,
-} from "@salt-ds/lab";
-import { ComponentMeta, Story } from "@storybook/react";
+import { Button, Text } from "@salt-ds/core";
+import { Tooltip, TooltipProps } from "@salt-ds/lab";
+import { ComponentMeta, ComponentStory, Story } from "@storybook/react";
 import { useCallback } from "react";
 
 export default {
   title: "Lab/Tooltip",
   component: Tooltip,
+  argTypes: {
```

Comment:
Why are we manually defining the arg types?

Reply:
Mostly to set the options, so people can have a play with the props. Should I remove them?

Reply:
I have a feeling at least for the simple ones like booleans, they will be there even if we don't manually set them here, worth deleting them and checking if there is any that needs to be manually added, otherwise we can delete them

Reply:
Interestingly they seem to be set after I've removed the argTypes, but I've purposely added them because the options were not displayed. 
Anyway...I've removed them. Hope it'll still work when it gets re-deployed. Thanks!